geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/stats/RangeHistogram.scala

```diff
@@ -155,14 +155,14 @@ object RangeHistogram {
 
   def buffer[T](value: T): (T, T) = {
     val buf = value match {
-      case v: String => (v, v + "z")
-      case v: Int    => (v - 1, v + 1)
-      case v: Long   => (v - 1, v + 1)
-      case v: Float  => (v - 1, v + 1)
-      case v: Double => (v - 1, v + 1)
-      case v: Date   => (v, new Date(v.getTime + 60000))
+      case v: String => (v + "0", v + "z")
```

Comment:
Do we have a pre-defined constant for the alphabet for this base conversion?  If so, can we reference it here?  Otherwise, it might takes folks a minute to realize where the "0" and "z" characters come from / what they represent.  Just a suggestion.